integration_tests/testsuite/test_util.py

```diff
@@ -75,10 +90,10 @@ def _get(url, timeout=DEFAULT_TIMEOUT):
     logging.info('making get request to url {0}'.format(url))
     try:
         response = requests.get(url)
-        return response.content, _check_response(response,
-                                                 'error when making get ' +
-                                                 'request! url: {0}'
-                                                 .format(url))
+        return _check_response(response,
```

Comment:
This swap doesn't seem to simplify much, any particular reason for it? Single return values are usually a bit easier to understand.